dash/dash.py

```diff
@@ -711,11 +728,28 @@ def serve_component_suites(self, package_name, path_in_package_dist):
             package.__path__,
         )
 
-        return flask.Response(
+        response = flask.Response(
             pkgutil.get_data(package_name, path_in_package_dist),
             mimetype=mimetype,
         )
 
+        if fingerprint:
```

Comment:
🐮  I found that the variable assignment before is confusing, and it might be better to put the `res is not None` directly here.

Reply:
sure